test/manifests/ingress-controller/mandatory.yaml

```diff
@@ -0,0 +1,283 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: ingress-nginx
```

Comment:
Should we just use the kube-system namespace that minikube already provides.

Reply:
The idea is to use the manifest we provide to the users in this repository and also be able to run e2e tests in any kubernetes cluster